src/lib/addons/slack.ts

```diff
@@ -82,6 +108,46 @@ export default class SlackAddon extends Addon {
         this.logger.info(`Handled event ${event.type}. Status codes=${codes}`);
     }
 
+    generateEnvironmentToggleText(event: IEvent): string {
+        const { environment, project, data, type } = event;
+        const toggleStatus =
+            type === FEATURE_ENVIRONMENT_ENABLED ? 'enabled' : 'disabled';
+        const feature = `<${this.featureLink(event)}|${data.name}>`;
+        return `The feature toggle *${feature}* in the ${project} project had the environment ${environment} ${toggleStatus}`;
+    }
+
+    generateStrategyChangeText(event: IEvent): string {
+        const { environment, project, data, type } = event;
+        const feature = `<${this.featureLink(event)}|${data.featureName}>`;
+        const strategyLink = `<${this.strategiesLink(event)}>`;
+        let action;
+        if (FEATURE_STRATEGY_UPDATE === type) {
+            action = 'updated';
+        } else if (FEATURE_STRATEGY_ADD) {
+            action = 'added';
+        } else {
+            action = 'removed';
+        }
+        const strategyText = `the strategy ${strategyLink} of type ${data.name} ${action} in the *${environment}* environment`;
+        return `The feature toggle *${feature}* in project: ${project} had ${strategyText}`;
+    }
+
+    generateMetadataText(event: IEvent): string {
+        const { createdBy, project, data } = event;
+        const feature = `<${this.featureLink(event)}|${data.name}>`;
+        return `${createdBy} updated the metadata for ${feature} in project ${project}`;
+    }
+
+    generateProjectChangeText(event: IEvent): string {
+        const { createdBy, project, data } = event;
+        return `${createdBy} moved ${data.name} to ${project}`;
+    }
+
+    strategiesLink(event: IEvent): string {
+        const { environment, project, data } = event;
+        return `${this.unleashUrl}/projects/${project}/features/${data.featureName}/environments/${environment}/strategies/${data.id}`;
```

Comment:
I do not think you can link directly to a strategy config. Should we? @FredrikOseberg ?

Reply:
Well, you can, but that link would probably be the api url rather than the frontend URL. Good point

Reply:
So what should we do? Just replace the link with `${data.name} was ${action} to the *${envrironment}* environment in project *${project}*`